pytest_jira.py

```diff
@@ -337,8 +337,9 @@ def pytest_addoption(parser):
     # FIXME - Change to a credentials.yaml ?
     config = six.moves.configparser.ConfigParser()
     config.read([
-        '/etc/jira.cfg',
-        os.path.expanduser('~/jira.cfg'),
+        str(os.path.join('/', 'etc', 'jira.cfg')),
```

Comment:
I believe that `str()` around `os.path.*` is not necessary in all three lines. Since it always return `str` or raise exception on bad input values.

Reply:
It actually returns a *PathLike* object, which has a \_\_str\_\_ dunder. I agree that it is unnecessary, but it seemed to me that your tests might disagree. I will try and change it.

Reply:
What version of python are you using ?
I was playing with py3.7.2, and I see `str` there. it might differ between versions ...

```
Python 3.7.2 (default, Jan 16 2019, 19:49:22) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import os.path

In [2]: type(os.path.join('/aaa', 'bbb'))
Out[2]: str
```